src/lib/selectCylinderSets.ts

```diff
@@ -0,0 +1,37 @@
+import { Knex } from 'knex';
+import { Cylinder, CylinderSet } from '../types/cylinderSet.types';
+import { log } from './log';
+
+export default async function selectCylinderSets(
+  trx: Knex.Transaction
+): Promise<CylinderSet[] | undefined> {
+  const set: CylinderSet[] | undefined = await trx(
+    'diving_cylinder_set'
+  ).select<CylinderSet[]>('id', 'owner', 'name');
```

Comment:
```suggestion
  const set: CylinderSet[] = await trx<CylinderSet[]>(
    'diving_cylinder_set'
  ).select('id', 'owner', 'name');
```
Tällä korjauksella kait vältetään se, että tietokantakyselyn paluutyyppi voisi olla undefined. Ei mitään käryä missä tilanteessa se palauttaisi undefined.... taulun puuttuminen ainakin johtaa ihan poikkeukseen.